### DISCUSSION TITLE
Weights for model sequential have not yet been created

### DISCUSSION THREAD

**Discussion by ParnianA** [Rishit-dagli/GLOM-TensorFlow#21] [Q&A] [answered]
Hi Rishit,

Even after calling the model with _model(img, iters = 12, return_all = True)_ I get the following error when I want to get trainable_weights:

_Weights are created when the Model is first called on inputs or `build()` is called with an `input_shape`._

Do you have any solutions for  this? Thanks, Parnian

**Comment by Rishit-dagli** [comment, marked as answer]
Hi @ParnianA ,
I wasn't able to reproduce this error, see this [Colab Example](https://colab.research.google.com/github/Rishit-dagli/GLOM-TensorFlow/blob/example/examples/glom_example.ipynb). Maybe some more info would be helpful.
